client/my-sites/upgrades/checkout-thank-you/jetpack-plan-details.jsx

```diff
@@ -49,13 +55,17 @@ const JetpackPlanDetails = ( { selectedSite } ) => {
 			);
 			props.buttonText = i18n.translate( 'Installation Instructions' );
 			props.href = 'https://en.support.wordpress.com/setting-up-premium-services/';
+			props.onClick = trackManualInstall( 'calypso_plans_autoconfig_click_manual_install' );
```

Comment:
maybe we can skip the `trackManualInstall` factory function just calling 
```
props.onClick = analytics.tracks.recordEvent.bind( analytics.tracks, 'calypso_plans_autoconfig_click_manual_install' );
```

I haven't tried it (yet) but I think that should work